### PR TITLE
Use the correct repo for minisat

### PR DESCRIPTION
**Pull request by hedtke**
Specify library name and version:  **minisat/2.2.0**

PR #16071 uses the wrong repository

this PR also
- fixes the missing `transitive_libs` option (see https://github.com/conan-io/conan-center-index/pull/16071#discussion_r1155320113) and
- enabled Windows builds with Ninja (see https://github.com/conan-io/conan-center-index/pull/16071#issuecomment-1493901117)

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by RubenRBS — APPROVED** [review]
Thanks for the followup, sorry that I did not notice the errors, nice catch @SpaceIm :)

If you could initialize an object that would have failed without the `transitive_libs` options so we make sure this does not happen again, that would be great!

**Comment by SpaceIm** [comment]
I'll submit a PR against your branch, it can be far simpler and avoid to change header files.

**Comment by SpaceIm** [comment]
please consider https://github.com/hedtke/conan-center-index/pull/1

I've not tested but it should allow to build msvc shared as well.

File: recipes/minisat/all/conanfile.py [hunk 1]
```diff
@@ -44,45 +36,34 @@ def configure(self):
...
+        cmake = CMake(self)
+        cmake.install()
 
     def package_info(self):
-        self.cpp_info.libs = collect_libs(self)
-        if self.settings.os in ["Linux", "FreeBSD"]:
-            self.cpp_info.system_libs.append("m")
-            self.cpp_info.system_libs.append("pthread")
+        self.cpp_info.libs = ["minisat"]
+        self.cpp_info.includedirs.append(os.path.join("include", "minisat"))
```
Comment (SpaceIm) [inline recipes/minisat/all/conanfile.py:69]:
```suggestion
        self.cpp_info.includedirs.append(os.path.join("include", "minisat"))
        if self.settings.os in ["Linux", "FreeBSD"]:
            self.cpp_info.system_libs.append("m")
```

so libm is required, but not pthread

**Review by SpaceIm — APPROVED** [review]

**Review by RubenRBS — APPROVED** [review]

**Pull request merged**